content/agent/proxy.md

```diff
@@ -29,22 +29,66 @@ A few options are available to send traffic to Datadog over SSL/TLS for hosts th
 
 Traditional web proxies are supported natively by the Agent. If you need to connect to the Internet through a proxy, edit your Agent configuration file.
 
-##### Agent v6
+### Agent v6
+
+Set different proxy servers for `https` and `http` requests in your Agent `datadog.yaml` configuration file. 
+The Agent uses `https` to send data to Datadog, but integrations might use `http` to gather metrics. No matter the proxied queries, you can activate SSL on your proxy server. Below are some configuration examples for your `datadog.yaml` file:
+
+Setting an HTTP proxy for all `https` queries:
```

Comment:
Not sure it's relevant to mention it in this document, but it could be since I've heard confusion about this from a lot of engineers (including a lot at Datadog): when setting up an HTTP proxy for `https` requests, the actual communication between the Agent and Datadog _is_ encrypted end-to-end with TLS and _cannot be_ decrypted by the proxy. The only unencrypted communication is the `HTTP CONNECT` request that's made between the Agent and the proxy to establish the initial tcp connection between the Agent and Datadog. As such, when using a proxy for `https` requests, there is no need to use an HTTPS proxy in order to have encrypted communication between the Agent and DD.

I'm mentioning this because even though most customers seem to use regular HTTP proxies (i.e. that only accept non-TLS `HTTP CONNECT` requests), some of them customers still try to configure an `https://` proxy URL under `proxy: https:` because they assume it's required/more secure for https requests, and it doesn't work, where instead they should use an `http://` proxy URL.

Anyway, in terms of changes to this doc, I'd suggest adding a note on the safety of using an HTTP proxy for HTTPS requests, and mentioning under the HTTPS proxy example that it should only be used if the proxy supports establishing HTTPS connections with clients for the initial `HTTP CONNECT` request.